src/main.py

```diff
@@ -152,7 +198,17 @@ async def help(interaction: Interaction):
 # Ignore non-slash commands
 @client.event
 async def on_message(message: Message):
-    pass
+    if (
+        client.user.mentioned_in(message) or "=duck" in message.clean_content
+    ) and message.author != client.user:
+        attachment = message.attachments[0] if message.attachments else None
+
+        bot_response = await client.gemini_model.query(
+            author=message.author,
+            message=message.clean_content.replace("=duck", ""),
+            attachment=attachment,
+        )
+        await message.reply(embeds=bot_response, mention_author=False)
```

Comment:
Change `=duck` to `=chat` so it's clear that it's actually a chat related command.